unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Resources/NavMap.prefab

```diff
@@ -1078,7 +1078,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
```

Comment:
I would delete the button alltogether.

Reply:
My bad, I forgot to push the deleted button prefab. Doing it now ;)